.vscode/cmake-kits.json

```diff
@@ -0,0 +1,16 @@
+[
+    {
+        "name": "native-gcc",
+        "toolchainFile": "${workspaceFolder}/verification/cpp/cmake/toolchains/gcc-native.cmake",
+        "cmakeSettings": {
+            "LIBUAVCAN_FLAG_SET": "../cmake/compiler_flag_sets/native.cmake"
```

Comment:
How did Libuavcan creep into this? I understand this is probably just carried over from your Libuavcan project setup, but if you want it kept in the public Nunavut repo then it's best to find a better name to avoid undesirable association between Nunavut and Libuavcan. If I understand this thread correctly, they should not be interdependent: https://forum.uavcan.org/t/library-independent-code-generation/589

Reply:
Cruft. I'll rename this. There is no libuavcan dependency.